front/app/components/admin/GraphCards/RegistrationsByTimeCard/Chart.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useMemo } from 'react';
+
+// styling
+import { MARGINS } from 'components/admin/GraphCards/_utils/style';
+
+// components
+import LineBarChart from 'components/admin/Graphs/LineBarChart';
+
+// utils
+import { isNilOrError, NilOrError } from 'utils/helperUtils';
+import { toThreeLetterMonth } from 'utils/dateUtils';
+import { generateEmptyData } from './generateEmptyData';
+
+// typings
+import {
+  Dates,
+  Resolution,
+  Layout,
+  ProjectId,
+} from 'components/admin/GraphCards/typings';
+import { TimeSeries } from './useRegistrationsByTime/typings';
+
+type Props = ProjectId &
+  Dates &
+  Resolution & {
+    timeSeries: TimeSeries | NilOrError;
+    innerRef: React.RefObject<any>;
+    layout?: Layout;
+  };
+
+const Chart = ({
+  projectId,
+  timeSeries,
+  startAtMoment,
+  endAtMoment,
+  resolution,
+  innerRef,
+  layout = 'wide',
+}: Props) => {
+  const emptyData = useMemo(
+    () => generateEmptyData(startAtMoment, endAtMoment, resolution),
+    [startAtMoment, endAtMoment, resolution]
+  );
+
+  const formatTick = (date: string) => {
+    return toThreeLetterMonth(date, resolution);
+  };
+
+  // Avoids unmounted component state update warning
+  if (timeSeries === undefined) {
+    return null;
+  }
+
+  const noData = isNilOrError(timeSeries) || !!projectId;
```

Comment:
Actually I think we can completely remove this `projectId` here, no? We're not using it for anything (unless I'm missing something)

Reply:
i use it only here copying the behaviour of active users in the visitors dashboard being blank when a project filter is applied because the filter is not relevant here.

Reply:
Ah wait, this is registrations, and I guess registrations only make sense for the platform level. So then I guess you could either do `!projectId` (if a filter is selected: show nothing) or just remove the `projectId` stuff (ignore filter, always show data for whole platform). I think maybe the former is better